EIPS/eip-7907.md

```diff
@@ -39,6 +39,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 `COLD_ACCOUNT_ACCESS_COST` and `WARM_STORAGE_READ_COST` are defined in [EIP-2929](./eip-2929.md#parameters).
 
 3. Update the [EIP-3860](./eip-3860.md) contract initcode size limit of 48KB (`0xc000` bytes) to 512KB (`0x80000` bytes).
+4. If a large contract is the entry point of a transaction, the cost calculated in (2) is charged before the execution of any opcode and contract code becomes warm immediately. The execution can run out-of-gas at this point if not enough gas was provided with the transaction to pay for this extra cost.
```

Comment:
Would be good to specify what would happen with balance transfered (maybe reverted? Not sure what is best). And that this gas doesn't get included in initial gas fee.